### PR TITLE
Added firebase stuff and a list of items

### PR DESCRIPTION
**Pull request by ldbonkowski**
Basic setup for firebase

### CODE REVIEW

File: .idea/compiler.xml
Comment (owencooke) [inline .idea/compiler.xml:1]: Do we need these .idea files? Or should we put a global .gitignore rule on all .idea files?

Comment (ldbonkowski) [reply .idea/compiler.xml:1]: I don’t think we need them, I just used the gitignore they had somewhere on eclass but I think we should be able to ignore the whole folder, I’ll throw that in tomorrow

**Comment by ldbonkowski** [comment]
image
Ugly but somewhat functional list with live db updates. Should be able to start work from here on the next tickets

File: app/src/main/java/com/example/househomey/Item.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        }
+        if (data.containsKey("serialNumber")) {
+            this.serialNumber = (String) data.get("serialNumber");
+        }
+        if (data.containsKey("comment")) {
+            this.comment = (String) data.get("comment");
+        }
+
+        this.cost = new BigDecimal((String) data.get("cost")).setScale(2);
+//        this.cost = new BigDecimal("2.00");
```
Comment (owencooke) [inline app/src/main/java/com/example/househomey/Item.java:45]: remove test value

File: app/src/main/java/com/example/househomey/ItemAdapter.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    @Override
+    public View getView(int position, @Nullable View convertView, @NonNull ViewGroup parent) {
+        View view = convertView;
+
+        // Inflate new view if there is nothing to recycle
+        if (view == null) {
+            view = LayoutInflater.from(context).inflate(R.layout.list_item, parent, false);
+        }
+
+        // Get the correct expense
```
Comment (owencooke) [inline app/src/main/java/com/example/househomey/ItemAdapter.java:38]: still says "expense". change to item

File: app/src/main/java/com/example/househomey/ItemList.java [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+    public void addItem(Item item) {
+        itemsRef.add(item.data());
+    }
+
+    public ArrayList<Item> getItems() {
+        return items;
+    }
+/*
+        db = FirebaseFirestore.getInstance();
```
Comment (owencooke) [inline app/src/main/java/com/example/househomey/ItemList.java:56]: is this just for reference of other functions? Do we need it?

**Review by owencooke — COMMENTED** [review]
Overall, looks great to me for a starting point. Added some nitpicks to clean up a little

**Review by owencooke — APPROVED** [review]

**Review by jdrco — APPROVED** [review]

**Pull request merged**